src/services/DfusionService.ts

```diff
@@ -289,6 +302,24 @@ export class DfusionRepoImpl implements DfusionService {
         .toNumber(),
     )
   }
+
+  private async _getTcr(): Promise<Set<string>> {
```

Comment:
Good enough! I', find with thios, although I prefer to use a library like the one we used in dex-react

Reply:
Yeah, was feeling too lazy to add a new dependency.
I'll do it in another PR and also cache the erc20 token details.

Reply:
https://github.com/gnosis/dex-telegram/issues/199